lib/ansible/inventory/manager.py

```diff
@@ -609,6 +609,8 @@ def subset(self, subset_pattern):
             # allow Unix style @filename data
             for x in subset_patterns:
                 if x[0] == "@":
+                    if not os.path.exists(x[1:]):
```

Comment:
might want to add to_bytes()

Reply:
I'd probably also save `x[1:]` in a temp var and use that instead of doing slicing multiple times.